LayoutTests/js/object-literal-computed-methods-expected.txt

```diff
@@ -18,7 +18,7 @@ PASS o.add(42, -10) is 32
 PASS typeof o.add is 'function'
 PASS o.add.length is 2
 PASS o.add.name is 'add'
-PASS o.add.toString() is 'function (x, y) { return x + y; }'
+PASS o.add.toString() is '[methodName](x, y) { return x + y; }'
```

Comment:
As weird as those two guys looked, that's how I understand the spec. 

Also tested on SpiderMonday and V8 to cross-check, and they behaves the same as this:

```
> o = { ['f'+'oo']() { return 10; } };
> o.foo.toString()
"['f'+'oo']() { return 10; }"
> methodName = 'add'; o = { [methodName](x, y) { return x + y; } };
> o.add.toString()
'[methodName](x, y) { return x + y; }'